packages/client/README.md

```diff
@@ -1,60 +1,52 @@
 <p align="center">
```

Comment:
Can we revert the changes here? Unsure what the large diff is for.

Reply:
Absolutely, not sure why it's diffing entire file, the content is the exact same...